website/content/docs/waypoint-hcl/variables/input.mdx

```diff
@@ -366,9 +366,19 @@ so the required environment variable name will usually have a mix of upper
 and lower case letters as in the above example.
 
 -> **Note:** Environment variables specified with `env` are only loaded
-as default values on the Waypoint [runner](/waypoint/docs/runner). For local operations
-this is the same machine as the CLI but for production use cases this might
-be a remote machine.
+as default values on the Waypoint [runner](/waypoint/docs/runner) and not the
+on demand runners. For local operations this is the same machine as the CLI
+but for production use cases this might be a remote machine.
```

Comment:
```suggestion
as default values on the Waypoint [runner](/waypoint/docs/runner).
For local operations this is the same machine as the CLI
but for production use cases this might be a remote machine.
```

It should be true that `env` will be picked up by the on-demand runners

Reply:
oh right, I think I was confusing this with our previous discussions about it.